projects/storefrontlib/src/cms-components/product/product-tabs/product-reviews/product-reviews.component.html

```diff
@@ -114,7 +123,6 @@ <h3>{{ 'productReview.overallRating' | cxTranslate }}</h3>
             type="submit"
             class="btn btn-block btn-primary"
             [ngClass]="{ 'submit-btn': reviewForm.valid }"
```

Comment:
The validation wasn't removed intentionally. We had to get rid of `disabled` attribute. We have to bring back the validation. A ticket for this will be created.

Reply:
We brought validation back